packages/lisk-transaction-pool/src/transaction_pool.ts

```diff
@@ -198,14 +213,37 @@ export class TransactionPool {
 		this._queues.verified.enqueueMany(transactions);
 	}
 
+	// It is assumed that signature is verified for this transaction before this function is called
```

Comment:
we don't need this comment as transaction.addSignature will validate transaction too